### ISSUE TITLE
Opening external url

### ISSUE THREAD

**Issue by Xenius97** [multitheftauto/mtasa-blue#3758]
### Is your feature request related to a problem? Please describe.

A function to open an URL in external browser.
When function is called, MTA goes automatically minimized and URL should open in default browser

### Describe the solution you'd like

```lua
openExternalURL('https://google.com/') -- this will open google.com in windows default browser
```

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not about a cheat or security vulnerability.

**Comment by MegadreamsBE** [comment]
Huge security risk. This should never be introduced

**Comment by TracerDS** [comment]
> Huge security risk. This should never be introduced

Unless we do the same what discord does: dialog for the user to either accept it or refuse it

**Comment by Xenius97** [comment]
> Huge security risk. This should never be introduced

I don't think so, if user need to accept it like requestBrowserDomains()
or only allow whitelisted domains

**Comment by MegadreamsBE** [comment]
> > Huge security risk. This should never be introduced
> 
> Unless we do the same what discord does: dialog for the user to either accept it or refuse it

True. We'll need to make sure to display the **entire** URL, and a clear warning that it can be dangerous.

> > Huge security risk. This should never be introduced
> 
> I don't think so, if user need to accept it like requestBrowserDomains() or only allow whitelisted domains

Combination between this and Tracer's proposal perhaps. Although the dialog Tracer proposes should be sufficient. Whitelisting an entire domain, and then just letting it happen at any point thereafter would still be a bad idea.

**Comment by PlatinMTA** [comment]
This was requested before... i hope this time we actually allow it

we just need to prompt the user before opening the url is not that problematic

**Comment by Fernando-A-Rocha** [comment]
The web is full of url redirects. I think having this with a prompt before opening the url is more than enough

**Comment by Fernando-A-Rocha** [comment]
Related to previously rejected feature: #2409

**Comment by CrosRoad95** [comment]
> Huge security risk. This should never be introduced

Skill issue, add prompt showned every time you try do this + require first to add url to whitelist. - discord can do this, you can too

And whitelist should be per server ( downloaded resources too ) because of https://developer.chrome.com/blog/http-cache-partitioning?hl=en

The same should apply for my pr with drag and drop onto mta window, add prompt, configuration ( by default disabled ).

If someone is stupid he get hacked like recently i even reported server to hosting provider due to owner stupidity.

I would personally use it to open website to documentation of my server

AND ALSO, if we talking about security risk, why noobody over the years configured some tool to find memory leak, to find where mta write/read outside buffers?

**Comment by Inder00** [comment]
Closed in #2409

**Comment by botder** [comment]
This was rejected before: https://github.com/multitheftauto/mtasa-blue/pull/2409#issuecomment-939493863

**Comment by Xenius97** [comment]
> This was rejected before: [#2409 (comment)](https://github.com/multitheftauto/mtasa-blue/pull/2409#issuecomment-939493863)

CEF could open dangerous URLs too, so based on this, that could also be removed.
